docs/ref/settings.txt

```diff
@@ -371,6 +371,33 @@ Whether to use a secure cookie for the CSRF cookie. If this is set to ``True``,
 the cookie will be marked as "secure," which means browsers may ensure that the
 cookie is only sent with an HTTPS connection.
 
+.. setting:: CSRF_USE_SESSIONS
+
+CSRF_USE_SESSIONS
+-----------------
+
+.. versionadded:: 1.11
+
+Default: ``False``
+
+Whether to store the CSRF token inside a the user's session instead of
+a cookie. This requires that you are using Django's session module, see
+:mod:`django.contrib.sessions` for more information.
+
+.. setting:: CSRF_SESSION_KEY
+
+CSRF_SESSION_KEY
+----------------
+
+.. versionadded:: 1.11
+
+Default: ``'csrftoken'``
+
+The name of the session key to use for the CSRF authentication token if
+:setting:`CSRF_USE_SESSIONS` is activated. . This can be whatever you want
```

Comment:
This can be any valid key

(At least with some session backends, keys need to be strings)